frontend/apps/reader/modules/readerstatus.lua

```diff
@@ -19,7 +19,6 @@ local ReaderStatus = WidgetContainer:extend{
 function ReaderStatus:init()
     if self.ui.document.is_pic then
         self.enabled = false
-        return
```

Comment:
:?

Reply:
Is it needed?

Reply:
PicDocument don't have a notion of status, yeah ;).

Reply:
So, we return at the end of init, without `return`.

Reply:
Oh, you're right, I'd missed the context.

Yeah, it's a bit clunky that way, my bad.

I was expecting something like

```lua
function ReaderStatus:init()
    if self.ui.document.is_pic then
        self.enabled = false
        return
    end

    self.total_pages = self.document:getPageCount()
    self.ui.menu:registerToMainMenu(self)
end
```

Reply:
@hius07 
Thank you! This worked! 
Yes, the `return` should be removed, otherwise KOReader crashes. 

(I don’t have access to a PC right now so I just added those lines through Total Commander)